### PR TITLE
Minor grading fixes

### LINKED ISSUE #3090: Minor grading remarks
When writing the grading manual, I gathered these small remarks:

- In the stepper, a link to https://docs.dodona.be/nl/guides/teachers/grading/ can be added to the callout at the top of the page
- In the stepper cards, step 1 has a title in bold, step 2 has a title in h4, step 3 has no title. The title in step 2 is a question that is answered by one of the two buttons directly below, so it makes some sense to keep this. I would remove the "deadline" title of step 1.
- In step 3, it is not clear that you can skip adding grades
- The "how to evaluate a solution" callout on the evaluations overview was not updated for scoring.
- In the score overview, `user` should be removed from the top left cell
- For the individual exercises, we only show the score itself and not the maximum (i.e. "8" and not "8/10") but in the total column, we do show the maximum. If there's no reason for this, I would remove it there as well since it is cleaner.
- In the score overview, there is no distinction between the total column and the exercise columns. Adding a right vertical border to the right of total would make that clearer. Alternatives: slight bg color for the first 2 columns, putting the total score in bold.
- On the page where you grade an individual solution, the right card on the page has an extra top border
  image

### PR DESCRIPTION
**Pull request by chvp**
- [x] In the stepper, a link to https://docs.dodona.be/nl/guides/teachers/grading/ can be added to the callout at the top of the page
- [x] In the stepper cards, step 1 has a title in bold, step 2 has a title in h4, step 3 has no title. The title in step 2 is a question that is answered by one of the two buttons directly below, so it makes some sense to keep this. I would remove the "deadline" title of step 1.
- [x] In step 3, it is not clear that you can skip adding grades
- [x] The "how to evaluate a solution" callout on the evaluations overview was not updated for scoring.
- [x] In the score overview, `user` should be removed from the top left cell
- [x] For the individual exercises, we only show the score itself and not the maximum (i.e. "8" and not "8/10") but in the total column, we do show the maximum. If there's no reason for this, I would remove it there as well since it is cleaner.
- [x] In the score overview, there is no distinction between the total column and the exercise columns. Adding a right vertical border to the right of total would make that clearer. Alternatives: slight bg color for the first 2 columns, putting the total score in bold.
- [x] On the page where you grade an individual solution, the right card on the page has an extra top border

Closes #3090.

### CODE REVIEW

**Comment by chvp** [comment]
I'm not a huge fan of the titles of the different steps now:
screenshot_2021-09-11-204700

Feel free to push changes, I've been trying not to open my laptop on sundays recently.

File: app/assets/stylesheets/components/evaluations.css.scss [hunk 1]
```diff
@@ -201,6 +201,10 @@ $user-column-width: 170px;
     overflow: hidden;
   }
 
+  td.total-column {
+    border-right: 1px solid $border-color;
+  }
```
Comment (bmesuere) [inline app/assets/stylesheets/components/evaluations.css.scss:206]:
```suggestion
  .total-column {
    border-right: 2px solid $border-color;
  }
```

File: config/locales/views/evaluations/en.yml [hunk 2]
```diff
@@ -5,9 +5,8 @@ en:
...
+      explanation_yes_grading_html: "<p>In the summary table, you can click on the <i class='mdi mdi-chevron-right mdi-18'></i> icon next to an exercise. You will then be taken to an incompleted submission of a random student. If you wish, you can leave feedback on the code and assign scores using the score items you configured. Note that you can also assign scores without leaving feedback. You can not give scores to exercises without submissions, these count for 0 in the grade overview.</p><p>As a reminder, students will <b>not</b> be able see the given feedback and scores until you click on the 'release feedback' button below.</p>"
```
Comment (bmesuere) [inline config/locales/views/evaluations/en.yml:9]:
```suggestion
      explanation_yes_grading_html: "<p>In the summary table, you can click on the <i class='mdi mdi-chevron-right mdi-18'></i> icon next to an exercise. You will then be taken to an incomplete submission of a random student. If you wish, you can leave feedback on the code and assign scores using the score items you configured. Note that you can also assign scores without leaving feedback. You can not give scores to exercises without submissions, these are automatically assigned a 0 in the grade overview.</p><p>As a reminder, students will <b>not</b> be able see the given feedback and scores until you click on the 'release feedback' button below.</p>"
```

File: config/locales/views/evaluations/nl.yml [hunk 3]
```diff
@@ -5,8 +5,8 @@ nl:
...
+      explanation_yes_grading_html: "<p>In de samenvattende tabel kan je naast een oefening op <i class='mdi mdi-chevron-right mdi-18'></i> klikken. Je komt dan op een nog te bekijken oplossing van een willekeurige student terecht. Vervolgens kan je indien gewenst feedback op de code geven en scores toekennen via de scoreonderdelen die je geconfigureerd hebt. Je kan de scores ook invullen zonder feedback achter te laten. Oefeningen zonder ingediende oplossing kan je geen scores geven, die zullen als 0 meetellen in het puntenoverzicht.</p><p>Ter herinnering, de studenten krijgen de gegeven feedback en scores <b>niet</b> automatisch te zien, daarvoor dien je hieronder op 'Feedback vrijgeven' te klikken.</p>"
```
Comment (bmesuere) [inline config/locales/views/evaluations/nl.yml:9]:
```suggestion
      explanation_yes_grading_html: "<p>In de samenvattende tabel kan je naast een oefening op <i class='mdi mdi-chevron-right mdi-18'></i> klikken. Je komt dan op een nog te bekijken oplossing van een willekeurige student terecht. Vervolgens kan je indien gewenst feedback op de code geven en scores toekennen via de scoreonderdelen die je geconfigureerd hebt. Je kan de scores ook invullen zonder feedback achter te laten. Oefeningen zonder ingediende oplossing kan je geen scores geven, deze zullen automatisch een 0 krijgen in het puntenoverzicht.</p><p>Ter herinnering, de studenten krijgen de gegeven feedback en scores <b>niet</b> automatisch te zien, daarvoor dien je hieronder op 'Feedback vrijgeven' te klikken.</p>"
```

**Review by bmesuere — APPROVED** [review]
:+1: some suggestions below

File: config/locales/views/evaluations/nl.yml [hunk 4]
```diff
@@ -23,7 +23,7 @@ nl:
       pick_deadline: Deadline kiezen
       create_evaluation: Reeks evalueren
       explanation_title: "Wat is een evaluatie?"
-      explanation_html: "<p>Het evalueren van een reeks is een gestructureerde manier om de ingediende oplossingen van een reeks te bekijken en er feedback op te geven. Dodona houdt bij welke oplossingen al bekeken zijn zodat je het evaluatiewerk eenvoudig kan verdelen over meerdere lesgevers. De feedback die je geeft is niet onmiddelijk zichtbaar voor de studenten; je kiest zelf wanneer je deze vrijgeeft.</p>"
+      explanation_html: "<p>Het evalueren van een reeks is een gestructureerde manier om de ingediende oplossingen van een reeks te bekijken en er feedback op te geven. Dodona houdt bij welke oplossingen al bekeken zijn zodat je het evaluatiewerk eenvoudig kan verdelen over meerdere lesgevers. De feedback die je geeft is niet onmiddelijk zichtbaar voor de studenten; je kiest zelf wanneer je deze vrijgeeft.</p><p>Een handleiding over verbeteren kan vinden op <a href=\"https://docs.dodona.be/nl/guides/teachers/grading/\">onze documentatiesite</a>.</p>"
```
Comment (niknetniko) [inline config/locales/views/evaluations/nl.yml:26]:
```suggestion
      explanation_html: "<p>Het evalueren van een reeks is een gestructureerde manier om de ingediende oplossingen van een reeks te bekijken en er feedback op te geven. Dodona houdt bij welke oplossingen al bekeken zijn zodat je het evaluatiewerk eenvoudig kan verdelen over meerdere lesgevers. De feedback die je geeft is niet onmiddellijk zichtbaar voor de studenten; je kiest zelf wanneer je deze vrijgeeft.</p><p>Een handleiding over verbeteren kan je vinden op <a href=\"https://docs.dodona.be/nl/guides/teachers/grading/\">onze documentatiesite</a>.</p>"
```

File: config/locales/views/evaluations/en.yml [hunk 5]
```diff
@@ -5,9 +5,8 @@ en:
...
+      explanation_yes_grading_html: "<p>In the summary table, you can click on the <i class='mdi mdi-chevron-right mdi-18'></i> icon next to an exercise. You will then be taken to an incomplete submission of a random student. If you wish, you can leave feedback on the code and assign scores using the score items you configured. Note that you can also assign scores without leaving feedback. You can not give scores to exercises without submissions, these are automatically assigned a 0 in the grade overview.</p><p>As a reminder, students will <b>not</b> be able see the given feedback and scores until you click on the 'release feedback' button below.</p>"
```
Comment (niknetniko) [inline config/locales/views/evaluations/en.yml:9]:
```suggestion
      explanation_yes_grading_html: "<p>In the summary table, you can click on the <i class='mdi mdi-chevron-right mdi-18'></i> icon next to an exercise. You will then be taken to an incomplete submission of a random student. If you wish, you can leave feedback on the code and assign scores using the score items you configured. Note that you can also assign scores without leaving feedback. You can not give scores to exercises without submissions, these are automatically assigned a 0 in the grade overview.</p><p>As a reminder, students will <b>not</b> be able see the given feedback and scores until you click on the 'Release feedback' button below.</p>"
```

**Review by niknetniko — APPROVED** [review]

**Pull request merged**